app/views/events/forms/_Tournament.html.erb

```diff
@@ -6,7 +6,13 @@
   <%= form.label :game_mode, class: 'control-label col-lg-2' %>
   <div class="col-lg-10">
     <%= form.select(:game_mode, Tournament.game_modes.map {|k, _| [Tournament.human_game_mode(k), k]},
-                 { selected: @event.game_mode, include_blank: t('events.new.select_game_mode')},
-                 { class: 'form-control' }) %>
+                    { selected: @event.game_mode, include_blank: t('events.new.select_game_mode')},
+                    { class: 'form-control' }) %>
+  </div>
+</div>
+<div class="form-group" id="event_has_place_3_match">
+  <div class="col-lg-10">
```

Comment:
The label has to be outside the `col-lg-10`.

<img width="1287" alt="screen shot 2018-01-19 at 13 32 07" src="https://user-images.githubusercontent.com/1104693/35151101-3fd74b8a-fd1d-11e7-865a-d1c6a864d2d9.png">